docs/terminal/perth.md

```diff
@@ -13,12 +13,40 @@
 † *Non-standard positions* may only be used in accordance with [VATPAC Ratings and Controller Positions Policy](https://vatpac.org/publications/policies)
 
 ## Airspace
-The Vertical limits of the PH TCU are `SFC` to `F245`.     
-PH TCU is responsible for the Perth TCU, except:      
-R155A & B when **PEA TCU** is online from `A020` to `F160`. When R155A is active to `F160`, PH TCU airspace above R155A shall be released to **PEA TCU**  
+The PH TCU is responsible for the airspace within 36 DME of the PH VOR, `SFC` to `F245`. 
+
+When **PEA TCU** is online R155A & B is released to them from `A020` to `F160`.  When R155A is active to `F160`, PH TCU airspace above R155A shall be released to **PEA TCU**.
 
 JT CTR reverts to Class G when **JT ADC** is offline, and is administered by the relevant PH TCU controller.      
 
+## Runway Modes
+### Southwest Plan
+With the Southwest Plan active, all departures shall be assigned runway 21 by **PH ACD**. Arrivals shall be processed to either runway 21 or 24 based on their feeder fix, as per the table below:
+
+| Feeder Fix | Assigned Runway |
+| --- | --- |
+| JULIM | 21 |
+| CONNI | 21 |
+| WAVES | 21 |
+| BEVLY | 24 (or 21 if operationally required) |
+| GRENE | 24 |
+| SOLUS | 24 |
+
+### Northeast Plan
+With the Northeast Plan active, departures via `AVNEX`, `OTLED`, `MANDU`, `SOLUS`, and `KEELS` shall be assigned runway 03 by **PH ACD**. All other departures shall be assigned runway 06. All arrivals shall be processed to runway 03.
+
+## Scenic Flights
```

Comment:
Only information I've been able to find is that Voiceless coordination exists:

On the V65
- From TMA South to TMA North; Assigned at or below A015
- 'V65' is entered in label

On the V66
- From TMA South to TMA North; Assigned at or below A035
- 'V66' is entered in label

Presumably implying that any level can be assigned (including "Not above XXX"). But a level definitely must be assigned since its in Class C, and the level is not part of the coded clearance.

Reply:
Awesome, shall update